docs/storage-policies.md

```diff
@@ -39,6 +41,7 @@ method with the ID of the storage policy to assign and the ID of the user to whi
 > __Note:__ This method will check if an assignment already exists for the user and take appropriate action.
 > It should work regardless of the current status of the user.
 
+<!-- sample post_storage_policy_assignments -->
```

Comment:
This is the convenience method; you probably want to remove this tag since you already have the endpoint correctly tagged below

Reply:
Removed